Data/IDs.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AtO_Loader.Data;
+
+public class IDs
+{
+    /// <summary>
+    /// List of IDs of newly added cards.
+    /// </summary>
+    public static List<string> CustomCardIDsNew { get; set; } = new();
+}
```

Comment:
This class just holds a list of any modded cards with non-base game IDs (i.e. modded cards that aren't changes to existing cards in game).